action.yml

```diff
@@ -10,6 +10,10 @@ inputs:
     description: 'Message to prospective contributor'
     required: false
     default: ''
+  issueCurrentlyAssignedMessage:
+    description: 'Message to contributors if issue is already assigned'
+    required: false
+    default: ''
```

Comment:
I'd suggest a sensible default message so that users could update with minimal change.

Reply:
I think something like this:

"The issue you are trying to assign to your self is already assigned."

Reply:
added it to use that wording!